src/index.html

```diff
@@ -7,8 +7,59 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:regular,500"
+      rel="stylesheet"
+      >
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a class="logo"href="#homepage"><img src="images/logo.png"  alt="logo"></a>
```

Comment:
```suggestion
      <a class="logo"href="#homepage">
        <img src="images/logo.png" alt="logo">
      </a>
```
pay attention to code style